internal/backend/db/dbgorm/connections_test.go

```diff
@@ -46,11 +46,11 @@ func TestCreateConnectionForeignKeys(t *testing.T) {
 
 	// FIXME: ENG-571
 	// c.IntegrationID = &unexisting
-	// assert.ErrorContains(t, f.gormdb.createConnection(f.ctx, &c), "FOREIGN KEY")
+	// assertErrorContainsIgnoreCase(t, f.gormdb.createConnection(f.ctx, &c), "FOREIGN KEY")
 	// c.IntegrationID = nil
 
 	c.ProjectID = &unexisting
-	assert.ErrorContains(t, f.gormdb.createConnection(f.ctx, &c), "FOREIGN KEY")
+	assertErrorContainsIgnoreCase(t, f.gormdb.createConnection(f.ctx, &c), "FOREIGN KEY")
```

Comment:
Why not test for `ErrForeignKeyViolated` (from https://github.com/go-gorm/gorm/blob/master/errors.go)
Same for other checks below.

Reply:
it's not a gorm err. But sqlite and pg drivers err, So I cannot check for the gorm one

Reply:
so why is that error even there? is there a way that gorm can convert to it itself?

Reply:
see https://gorm.io/docs/error_handling.html#Dialect-Translated-Errors:
![image](https://github.com/autokitteh/autokitteh/assets/426603/aa6df9e9-0ddf-4f58-ac62-b659892a2c26)

Reply:
thanks, Itay! it works with error translation